pom.xml

```diff
@@ -51,6 +51,17 @@
 		<url>https://github.com/chrono24/stripes/actions</url>
 	</ciManagement>
 
+	<distributionManagement>
+		<snapshotRepository>
+			<id>ossrh</id>
+			<url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+		</snapshotRepository>
+		<repository>
+			<id>ossrh</id>
+			<url>https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+		</repository>
+	</distributionManagement>
```

Comment:
Why do we need these changes?

Reply:
Without them, the nexus plugin tries to upload to the legacy staging server https://oss.sonatype.org. I think this behavior must have changed with 1.6.9, which is kind of weird. Our account is a new one, and is only valid for the new staging server.